waspc/data/Generator/templates/sdk/wasp/client/operations/actions/core.ts

```diff
@@ -37,7 +37,10 @@ export function createAction<BackendAction extends GenericBackendOperation>(
   const action = (args) => internalAction(args, [])
   action.internal = internalAction
 
-  return action as ActionFor<BackendAction>
+  // todo(filip): why is unkonwn necessary here?
+  // Maybe something to do with
+  // https://www.typescriptlang.org/play/?#code/MYewdgzgLgBCBGArGBeGBvGBDAXDA5AGYgj4wC+AUAogHTyoHxYBO+llA9JzIQK5hgUAJbhsAG3EgA7hGxgYAUwBuIccuFgA5jCgBPAA6KY8PrBqKhMACYhFcsCCiVQkWPwVoAFAEpUAPgJiUkoPekZ8ZjYgA
```

Comment:
I'd slightly give more info in the comment (for your future self's sake). What is the error you are seeing? Maybe the playground can reproduce it more faithfully.

Reply:
I'll add the info somewhere (either here or in the playground).